manifest.json

```diff
@@ -21,6 +21,15 @@
     "default_popup": "popup/index.html"
   },
 
+  "commands": {
+    "_execute_browser_action": {
+      "suggested_key": {
+        "default": "Ctrl+Alt+L"
```

Comment:
I'm running Firefox on Pop!_OS. Happy to help see this through.

For Firefox, I basically checked using the browser search feature for a clashing shortcut, if any, on [a Firefox help article](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly).

Reply:
I wonder if `Ctrl+Alt+B` or `Ctrl+Alt+D` work better for Chrome? At least on Firefox, it isn't blocked by the browser and my password manager.

Reply:
Chrome doesn't allow any combination of `Ctrl+Alt` unfortunately:
> Note: All key combinations must include either Ctrl* or Alt. Combinations that involve Ctrl+Alt are not permitted in order to avoid conflicts with the AltGr key. Shift can be used in addition to Alt or Ctrl, but is not required. Modifiers (such as Ctrl) can not be used in combination with the Media Keys. Tab key was removed from list of supported keys in Chrome version 33 and above for accessibility reasons.

[See docs](https://developer.chrome.com/docs/extensions/reference/commands/) for more info.

Chrome uses `Ctrl+D` as default.

Reply:
So something with `Ctrl+Shift+...` would be appropriate for Chrome, however from my quick testing I could not get anything with `Shift` working in FF.

Reply:
I couldn't get `Ctrl+Shift` working either on Firefox.

I've tried `Alt+Shift+L` on both Firefox and Chromium and that works. I'd have preferred `Alt+Shift+B` since you can do that with one hand, but that is blocked by [Dark Reader's](https://darkreader.org/help/en/) defaults. On the other hand, Dark Reader allows for customizing its hotkey.

What do you think works better, then? `Alt+Shift+L` or `B`?

Reply:
Let's go with `Alt+Shift+L` then, tested that and works here as well.

Reply:
Sure. Perhaps we could work on making this a customizable hotkey in the future. :)